dockerfile/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ansible/ansible:ubuntu1604
+MAINTAINER zikalino
+
+# run apt-get update for later purposes
+RUN apt-get update
+
+# clone actual ansible directory as it's not a part of base ansible/ansible
+# setup all necessary dependencies
+RUN git clone https://github.com/zikalino/ansible.git --recursive
```

Comment:
i think we've synced that we'll install ansible from official package manger. not from repo. cause it's official ansible release. as our goal is to provide general ansible extension, it's better to use official release. if user really like to use some  devl release ansible, they could use your extension configuration to change docker image. any thoughts?